sources/tools/Stride.Importer.3D/Material/Materials.cs

```diff
@@ -160,5 +172,51 @@ public static unsafe MaterialStack ConvertAssimpStackCppToCs(Silk.NET.Assimp.Ass
 
             return ret;
         }
+
+        internal static unsafe (bool isOk, string errorMessage) TryGetTextureFileName(string materialTextureFileName, Scene* scene, out string fileName)
```

Comment:
The tuple return type for a Try[...] seems a bit awkward to work with;
```cs
var (isOk, errorMessage) = TryGetTextureFileName(elTexPath.AsString, scene, out texFileName);
if (!isOk)
{
    logger?.Error(errorMessage);
    continue; // error !
}
```
vs a try with another out var
```cs
if (!TryGetTextureFileName(elTexPath.AsString, scene, out texFileName, out var errorMessage))
{
    logger?.Error(errorMessage);
    continue; // error !
}
```
Not a big deal, but it's not standard in c#, and not sure it is any easier to read ?

Reply:
Yeah, I can never settle on the right way the TryGet pattern, haha.
I'll change it for returning `bool` and `out` the message since it's standard.

Reply:
Done, and swapped the logic in the TryGet methods so it doesn't need nested if statements.